header.php

```diff
@@ -30,6 +30,12 @@
 <head>
 <meta charset="<?php bloginfo( 'charset' ); ?>">
 <meta name="viewport" content="width=device-width, initial-scale=1">
+<?php
+// If set, configure 'featured image' as 'og:image'.
+if ( get_the_post_thumbnail_url( get_the_ID() ); ) {
+	echo esc_html( '<meta property="og:image" content='get_the_post_thumbnail_url( get_the_ID() )' />' )
```

Comment:
I think this will throw a syntax error. We should add the `.` concatenation operator for safety:
```suggestion
	echo esc_html( '<meta property="og:image" content="' . get_the_post_thumbnail_url( get_the_ID() ) . '" />' )
```

Reply:
> I think this will throw a syntax error. We should add the `.` concatenation operator for safety:

Ahh - thank you - will integrate and try out a bit more before pushing to production.